js/kucoinfutures.js

```diff
@@ -1144,6 +1144,12 @@ module.exports = class kucoinfutures extends kucoin {
         const rawCost = this.safeString2 (order, 'dealFunds', 'filledValue');
         const leverage = this.safeString (order, 'leverage');
         const cost = Precise.stringDiv (rawCost, leverage);
+        let average = undefined;
+        if (market['linear']) {
+            average = Precise.stringDiv (rawCost, Precise.stringMul (market['contractSize'], filled));
```

Comment:
What if `filled` is 0?

Reply:
Nevermind, should be ok.

Reply:
then average will be undefined for linear futueres and 0 for inverse futures

Reply:
I think we might need to set it to undefined for inverse futures, not 0.